_inc/jetpack-modules.views.js

```diff
@@ -19,7 +19,7 @@ window.jetpackModules.views = (function( window, $, _, Backbone, wp ) {
 					return;
 				}
 
-				var url      = window.location.href.split('?')[0] + '?page=jetpack_modules',
```

Comment:
Any objections to leaving `jetpack_modules` as the page?  I'm thinking for back-compat if people have some links already pointing to this.